torch/csrc/autograd/python_variable_indexing.cpp

```diff
@@ -175,7 +175,7 @@ static Variable applySlicing(const Variable& self, PyObject* index, variable_lis
     } else if (THPVariable_Check(obj)) {
       auto& var = THPVariable_Unpack(obj);
       auto scalar_type = var.scalar_type();
-      if (var.dim() == 0 && (at::isIntegralType(scalar_type) || scalar_type == ScalarType::Bool)) {
+      if (var.dim() == 0 && at::isIntegralType(scalar_type)) {
```

Comment:
is this change related?  Where does this come up?  Also shouldn't there be a warning somewhere?

Reply:
i added bool to isIntegralType so this part became redundant now